bazel/defs.bzl

```diff
@@ -291,6 +291,18 @@ def cc_firmware(name, **kwargs):
         data.append(btldr + ".hex")
         defines.append("BTLDR_ID=" + ECUS[btldr_ecu_name]["btldr_id"])
 
+    is_can_print = False
```

Comment:
just curious, what's the motivation behind also having cc_firmware be able to launch the print decoder? couldn't we just have a wrapper function that calls cc_firmware and the py_binary that you created to have the same effect?

Reply:
I was in the process of learning Bazel rules and started with an implementation that was less than optimal. I also wanted there to be no additional boilerplate in BUILD files. 

I think you're right - I should be able to add a wrapper that calls both. I'll give it another shot, but if it doesn't work the intended use of `can print` still works - that is run the decoder in a separate shell and flash firmware while developing.

Reply:
I'm remembering why bundling them together in a macro was difficult - passing some arguments to one rule and the rest to another rule isn't straightforward, at least in simple macros. I believe it's possible with more advanced Bazel features, but I think the simplest solution would instead be a `can print` config file.

For now I'll file in the errata until we run into issues or I learn more Bazel.

Reply:
sg